include/Core/Pages/PagesModule.php

```diff
@@ -21,4 +21,11 @@ public function filter_js_array( $js_array ) {
 
 		return $js_array;
 	}
+
+	/**
+	 * {@inheritDoc}
```

Comment:
Let's not add `@inheritDoc` since that is the default behaviour.

We are just adding noise without any value.